engine/modelc/scripts/test_model_importer.sh

```diff
@@ -59,7 +59,7 @@ if [ "Darwin" == "$(uname)" ]; then
     set -e
 
     PACKAGED_XCODE=${DYNAMO_HOME}/ext/SDKs/
-    PACKAGED_XCODE_TOOLCHAIN=${PACKAGED_XCODE}/Toolchains/XcodeDefault.xctoolchain
+    PACKAGED_XCODE_TOOLCHAIN=${PACKAGED_XCODE}/XcodeDefault16.2.xctoolchain
```

Comment:
> For backward compatability I've created a symlink with name Xcode{version}.xctoolchain in the root

If that's true, why the need to change these scripts. It feels we're making the script more brittle and easy to break?

Reply:
I'd recommend getting it from `sdk.py`: `sdk.PACKAGES_XCODE_TOOLCHAIN`

Reply:
Here is an example of usage:
https://github.com/defold/defold/blob/e93c7cc0724a6d847a646c7be4c1f8547742c95e/build_tools/set_sdk_vars.py#L21

Reply:
> > For backward compatability I've created a symlink with name Xcode{version}.xctoolchain in the root
> 
> If that's true, why the need to change these scripts. It feels we're making the script more brittle and easy to break?

Because `PACKAGED_XCODE_TOOLCHAIN=${PACKAGED_XCODE}/Toolchains/XcodeDefault.xctoolchain` doesn't exist